### ISSUE TITLE
Prison Break & Prison flyzone restriction

### ISSUE THREAD

**Issue by fresqo** [StrayaGamingTanoa/StrayaGaming.Tanoa#92]
- Create a prison break mission.. 
- Add no fly zone restriction

**Comment by envycam** [comment]
Yes do it, but concentrate on the other issues at hand, things like this and the military seize crate and the Marshall law button we can do after he core issues are fixed

**Comment by fresqo** [comment]
i propose a NEW prison, one that has a gym and church etc.. the 4 walls does my head in... we can do better imo
